src/components/gamma/split-view/SplitViewHost.types.ts

```diff
@@ -262,6 +264,28 @@ export interface SplitViewHostProps extends ViewProps {
    * @default automatic
    */
   preferredSplitBehavior?: SplitViewSplitBehavior;
+  /**
+   * @summary Specifies the background style of the primary view controller.
+   *
+   * On iOS 18 or lower, we always fall back to `none` which is a system default value. Since iOS 26 the system default value is `sidebar`.
+   *
+   * The following values are currently supported:
+   *
+   * - `default` - chosen by the OS, the appropriate background style is based on the device preferences
+   * - `none` - a style that has no visual effect on the background appearance of the primary view controller
+   * - `sidebar` - a style that applies a blurred effect to the background of the primary view controller
+   *
+   * The supported values corresponds to the official UIKit documentation:
```

Comment:
```suggestion
   * The supported values correspond to the official UIKit documentation:
```

also a nitpick but do those values correspond to UIKit docs if there is no `default` option there?